spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -210,6 +210,7 @@ public Object deserialize(byte[] bytes) throws ZkMarshallingError {
 
 	private static final Set<Object> KAFKA_PRODUCER_PROPERTIES = new SetBuilder()
 			.add(BinderPropertyKeys.MIN_PARTITION_COUNT)
+			.add(BinderPropertyKeys.REQUIRED_GROUPS)
```

Comment:
I'm not sure about how other binders can satisfy this, but if they cannot, they wouldn't be usable with SCDF.